src/main/java/com/phonebookservice/model/SocialProfile.java

```diff
@@ -0,0 +1,19 @@
+package main.java.com.phonebookservice.model;
+
+public class SocialProfile {
+    private String socialMediaName;
+    private String socialProfile;
+
+    /**
+     * social profile constructor.
+     *
+     * @param socialMediaName the social media name.
```

Comment:
Can you have SocialMediaName or should be limited options ? 

I think you should use Enum called SocialMediaType